test/eventing-contrib.bash

```diff
@@ -30,9 +30,6 @@ function upstream_knative_eventing_contrib_e2e {
 
   failed=0
 
-  logger.info 'Installing Strimzi'
-  (( !failed )) && install_strimzi || failed=$?
-
```

Comment:
- The function containing this block, `upstream_knative_eventing_contrib_e2e`, is only called by `upstream-e2e-tests.sh`
- `upstream-e2e-tests.sh` is only called in the makefile and we already install Strimzi in the makefile targets
- This block causes Strimzi to reinstall
- And the reinstall happens via the script in our eventing fork, which does not have the changes here for Strimzi in S-O

We already install Strimzi when we